buildSrc/src/main/kotlin/Project.kt

```diff
@@ -125,4 +135,5 @@ object Versions {
   const val mapboxSdkVersionsPlugin = "1.1.3"
   const val pitest = "0.2.8"
   const val detekt = "1.20.0"
+  const val compose = "1.1.0-beta03"
```

Comment:
I guess it's because of our kotlin version `1.5.31`

Reply:
yes, correct